runtime/lua/vim/ui.lua

```diff
@@ -140,11 +146,15 @@ function M.open(path)
   end
 
   local cmd --- @type string[]
-  local opts --- @type vim.SystemOpts
+  local conf --- @type vim.SystemOpts
 
-  opts = { text = true, detach = true }
+  conf = opts or {}
+  conf.text = opts.text or true
+  conf.detach = opts.detach or true
 
-  if vim.fn.has('mac') == 1 then
+  if conf.cmd then
+    cmd = vim.list_extend(conf.cmd, { path })
```

Comment:
looks like `opts` and `conf` are being mixed up. they aren't related. this should just be `opts.cmd`.

Reply:
see above